src/cli.rs

```diff
@@ -210,6 +210,42 @@ pub struct Opt {
     /// or PAGER (BAT_PAGER has priority).
     #[structopt(long = "paging", default_value = "auto")]
     pub paging_mode: String,
+
+    /// Display line numbers next to the diff. The first column contains line
+    /// numbers in the previous version of the file, and the second column contains
+    /// line number in the new version of the file. A blank cell in the first or
+    /// second column indicates that the line does not exist in that file (it was
+    /// added or removed, respectively).
+    #[structopt(short = "n", long = "number")]
+    pub show_line_numbers: bool,
+
+    /// Color for the left (minus) column of line numbers (--number), if --number is set.
+    /// Defaults to the configured hunk_color.
+    #[structopt(long = "number-minus-foreground-color")]
+    pub number_minus_foreground_color: Option<String>,
+
+    /// Color for the right (plus) column of line numbers (--number), if --number is set.
+    /// Defaults to the configured hunk_color.
+    #[structopt(long = "number-plus-foreground-color")]
+    pub number_plus_foreground_color: Option<String>,
+
+    /// Color for the right dividing line of the line numbers section, if --number is set.
+    /// Defaults to the configured hunk_color.
+    #[structopt(long = "number-divider-right-foreground-color")]
+    pub number_divider_right_foreground_color: Option<String>,
+
+    /// Color for the center dividing line of the line numbers section, if --number is set.
+    /// Defaults to the configured hunk_color.
+    #[structopt(long = "number-divider-center-foreground-color")]
+    pub number_divider_center_foreground_color: Option<String>,
+
+    /// String to use as the center divider of the line numbers section, if --number is set.
+    #[structopt(long = "number-divider-center-string", default_value = "⋮")]
+    pub number_divider_center_string: String,
+
+    /// String to use as the right divider of the line numbers section, if --number is set.
+    #[structopt(long = "number-divider-right-string", default_value = "│")]
+    pub number_divider_right_string: String,
```

Comment:
WDYT about changing these to `--number-right-divider-*` and `--number-center-divider-*`? (My brain parses "number-divider-right-foreground-color" as implying there's a right foreground.)

Reply:
Np, done! Also updated variable names to match.